eth_tester/utils/backend_testing.py

```diff
@@ -284,9 +284,16 @@ def test_send_raw_transaction_valid_raw_transaction(self, eth_tester, is_pending
         # assert that the raw transaction is confirmed and successful
         assert receipt['transaction_hash'] == transaction_hash
 
-    def test_send_raw_transaction_invalid_raw_transaction(self, eth_tester):
+    def test_send_raw_transaction_invalid_rlp_transaction(self, eth_tester):
         self.skip_if_no_evm_execution()
         invalid_transaction_hex = '0x1234'
+        import eth
+        with pytest.raises(eth.exceptions.UnrecognizedTransactionType):
+            eth_tester.send_raw_transaction(invalid_transaction_hex)
```

Comment:
Maybe this should wrap the exception in something like a `ValidationError`, although I think we've leaned pretty far away from actually supporting multiple backends.

Reply:
yeah, don't think we have the bandwidth to make multiple backends a priority either.